README.md

```diff
@@ -1,3 +1,5 @@
 # synthesized-insight
 
 [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/synthesized-io/insight/master.svg)](https://results.pre-commit.ci/latest/github/synthesized-io/insight/master)
+
+### Developers need to have python 3.8 installed because pre-commit uses python 3.8
```

Comment:
suggestion:

## Installation

### 1. Install python requirements:

#### macOS
(make sure you have brew installed)
```sh
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
```
```sh
brew install openssl readline sqlite3 xz zlib
```

#### ubuntu
```sh
sudo apt-get update; sudo apt-get install make build-essential libssl-dev zlib1g-dev \
libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
```

### 2. Install pyenv:

#### macOS
```sh
brew update
brew install pyenv
```

#### ubuntu
```sh
curl https://pyenv.run | bash
```

### 3. Install python 3.8.10:

```sh
pyenv install 3.8.10
```